docs/api-docs/getCmd.md

```diff
@@ -0,0 +1,11 @@
+# `getCmd(loop): Cmd | null`
+
+* `loop: any` &ndash; any object.
+* returns the cmd component of the array if the input is a `[any, cCmd]`
```

Comment:
cCmd is a typo I think. I would maybe say 

if the input is a `loop` object (`[any, Cmd]` pair)